src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  // Поділ  рядків крапками з комою
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    // Розділ кожну декларацію двокрапкою
+    const [property, ...values] = declaration.split(':');
+
+    // Перевірте на існ  власт, так і значн, і видал зайві пробіли
+    if (property && values.length > 0) {
+      styleObject[property.trim()] = values.join(':').trim();
```

Comment:
Using `trim` on both the property and value ensures that any extra whitespace is removed, which is crucial for the accuracy of the resulting object.